hastexo/hastexo.py

```diff
@@ -220,6 +220,11 @@ def parse_attributes(tag, node, block):
                         "environment": environment}
             block.providers.append(provider)
 
+    def to_bool(string_value):
```

Comment:
Any reason for rolling your own here, rather than using

```python
from distutils.util import strtobool

def to_bool(string_value):
  return bool(strtobool(value))
```

That approach would cover you for `true/TRUE/True/tRuE/yes/1` and `false/FALSE/False/FaLsE/0`.